app/models/webhook_endpoint.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class WebhookEndpoint < ApplicationRecord
+  LIMIT = 5
+
+  belongs_to :organization
+  has_many :webhooks, dependent: :destroy
+
+  validates :webhook_url, presence: true, url: true
+  validate :max_webhook_endpoints, on: :create
+
+  private
+
+  def max_webhook_endpoints
+    errors.add(:base, :exceeded_limit) if organization &&
+                                          organization.webhook_endpoints.reload.count >= LIMIT
```

Comment:
can't we do something like this: `organization.webhook_endpoints.count > LIMIT`?

Reply:
In that case it would add 6th webhook endpoint. The `max_webhook_endpoints` validation doesn't take into consideration the organization's webhook_endpoint that it's just being added, so another one is created.

You can try both implementations of `max_webhook_endpoints ` in rails console:

```
(WebhookEndpoint::LIMIT + 1).times { FactoryBot.create(:webhook_endpoint, organization: Organization.first) }
```